src/agent/__tests__/MessageSender.test.ts

```diff
@@ -0,0 +1,144 @@
+import testLogger from '../../__tests__/logger'
+import { MessageSender } from '../MessageSender'
+import { TransportSession, TransportService as TransportServiceImpl } from '../TransportService'
+import { EnvelopeService as EnvelopeServiceImpl } from '../EnvelopeService'
+import { createOutboundMessage } from '../helpers'
+import { AgentMessage } from '../AgentMessage'
+import { OutboundTransporter } from '../../transport'
+import { OutboundPackage } from '../..'
+import { OutboundMessage } from '../../types'
+import { ConnectionRecord } from '../../modules/connections'
+import { ReturnRouteTypes } from '../../decorators/transport/TransportDecorator'
+import { getMockConnection } from '../../__tests__/helpers'
+
+jest.mock('../TransportService')
+jest.mock('../EnvelopeService')
+
+const logger = testLogger
```

Comment:
maybe just `import logger from ../../__tests__/logger`?

Reply:
I like `const logger = testLogger` more because you can simply change it to `const logger = new ConsoleLogger(...)` when want to log into the console during the development.